__tests__/endpoint.test.js

```diff
@@ -78,4 +71,32 @@ describe('Endpoint Tests', () => {
               });
           });
     })
+    describe('/api/articles/', () => {
+        test.only('200: response with an articles array of article object, which should contain all collumns except body, and be in data descending order', () => {
+            return request(app).get('/api/articles').expect(200)
+            .then(({body}) => {
+                console.log(body.data, 'in test')
+                expect(body.data).toHaveLength(13)
+                expect(Array.isArray(body.data))
+                body.data.forEach((article) => {
+                    expect(article).toHaveProperty('author', expect.any(String))
+                    expect(article).toHaveProperty('title', expect.any(String))
+                    expect(article).toHaveProperty('article_id', expect.any(Number))
+                    expect(article).toHaveProperty('topic', expect.any(String))
+                    expect(article).toHaveProperty('created_at', expect.any(String))
+                    expect(article).toHaveProperty('votes', expect.any(Number))
+                    expect(article).toHaveProperty('article_img_url', expect.any(String))
+                    expect(article).toHaveProperty('comment_count', expect.any(String))
+                    expect(article).not.toHaveProperty('body')
+                })
+            })
+        })
+        test('GET:400 sends an appropriate status and error message when given an invalid path', () => {
```

Comment:
Just a heads up! We don't need to test a general bad path for **each** end point. This would just fall under a general bad path for any invalid end points (the possibilities for this are endless, so we don't need to test this for each end point)